package.json

```diff
@@ -8,9 +8,9 @@
 		"clean": "rimraf .next node_modules",
 		"lint": "next-hashicorp lint",
 		"lint:stylelint": "npx stylelint '**/*.css'",
-		"prebuild": "hc-tools ./scripts/generate-tutorial-variant-map.ts && hc-tools ./scripts/extract-hvd-content.ts",
```

Comment:
This is not longer needed as because we wait on the hvd content promise during SSR that promise already fires during build in prod and on demand during dev.

Reply:
love this for us